reflex/components/radix/themes/components/context_menu.py

```diff
@@ -232,6 +233,12 @@ class ContextMenuSeparator(RadixThemesComponent):
     tag = "ContextMenu.Separator"
 
 
+class ContextMenuCheckbox(HighLevelCheckbox):
+    """The component that contains the checkbox."""
+
+    tag = "ContextMenu.Checkbox"
+
```

Comment:
I think here we want the props `color` and `shortcut` as defined in the docs of radix UI.

Reply:
Did some testing on my side, and the `shortcut` doesn't seems to work, so let's just go for the `color_scheme` for now.